src/redispatch.js

```diff
@@ -29,9 +29,7 @@ const cancelEvent = (player, event) => {
 const prefixEvent = (player, prefix, event) => {
   cancelEvent(player, event);
   player.trigger({
-    type: prefix + event.type,
-    state: player.ads.state,
-    originalEvent: event
+    type: prefix + event.type
```

Comment:
we should probably continue to send the originalEvent

Reply:
I removed both `triggerevent` and `originalEvent` after careful consideration. My conclusion is that they are legacy debug code that should not remain part of the public interface.

Reply:
Hmm, let's leave it like this(removed) for now and discuss again before releasing

Reply:
After discussion adding back just originalEvent